app/components/mandatarissen/mandataris-card.hbs

```diff
@@ -1,116 +1,130 @@
-<AuCard as |c|>
-  <c.header>
-    <div class="au-u-flex">
-      <AuHeading @level="2" @skin="4" class="flex-grow">
-        {{this.rol}}
-      </AuHeading>
-      <AuPill
-        @skin="link"
-        @draft={{false}}
-        @size="regular"
-        @icon="info-circle"
-        @iconAlignment="left"
-        @hideText={{false}}
-      >
-        {{if
-          @mandataris.publicationStatus
-          @mandataris.publicationStatus.label
-          "Bekrachtigd"
-        }}
-      </AuPill>
-    </div>
-  </c.header>
-  <c.content>
+<AuToolbar @size="large" class="au-u-padding-none au-u-bottom-none" as |Group|>
+  <Group>
+    <AuHeading @skin="4">
+      Details mandaat
+    </AuHeading>
+  </Group>
+</AuToolbar>
 
-    <div class="au-o-grid au-o-grid--tiny">
-      <div class="au-o-grid__item au-u-1-2 au-u-1-3@medium">
-        <div class="au-c-description-label">Status</div>
-        <div class="au-c-description-value">{{this.status}}</div>
-      </div>
-      <div class="au-o-grid__item au-u-1-2 au-u-1-3@medium">
-        <div class="au-c-description-label">Rangorde</div>
-        <div class="au-c-description-value">{{@mandataris.rangorde}}</div>
-      </div>
-      <div class="au-o-grid__item au-u-1-2 au-u-1-3@medium">
-        <div class="au-c-description-label">Fractie</div>
-        <div class="au-c-description-value"><Mandaat::FoldedFracties
-            @persoon={{this.persoon}}
-            @bestuursperiode={{@bestuursperiode}}
-          /></div>
+<div class="au-o-box au-u-background-gray-100">
+  <div class="au-o-grid au-o-grid--tiny">
+    <div class="au-o-grid__item au-u-1-2">
+      <div class="au-c-description-label">Status</div>
+      <div
+        class="au-c-description-value au-u-margin-top-tiny"
+      >{{this.status}}</div>
+    </div>
+    <div class="au-o-grid__item au-u-1-2">
+      <div class="au-c-description-label">Publicatie status</div>
+      <div class="au-c-description-value au-u-margin-top-tiny">
+        <AuPill
+          @skin="link"
+          @draft={{false}}
+          @size="regular"
+          @icon="info-circle"
+          @iconAlignment="left"
+          @hideText={{false}}
+        >
+          {{if
+            @mandataris.publicationStatus
+            @mandataris.publicationStatus.label
+            "Bekrachtigd"
+          }}
+        </AuPill>
+        {{! <AuLinkExternal href="" class="au-u-margin-left-tiny">Bekijk bewijs</AuLinkExternal> }}
```

Comment:
why comment this? i'd rather we leave it in and put a todo on the href if that's what we're waiting for. Just add a bug ticket for it, it's super small and this way it's tracked

Reply:
Add ticket here LMB-481, LMB-482